portfolio/src/main/java/com/google/sps/servlets/Comment.java

```diff
@@ -0,0 +1,16 @@
+package com.google.sps.servlets;
+
+public class Comment {
+  long id;
+  String text;
+  String timestamp;
+
+  public Comment(){
+  }
+
+  public Comment(long id, String text, String timestamp){
+    this.id = id;
+    this.text = text;
+    this.timestamp = timestamp;
```

Comment:
I know that the Task example use the variable name timestamp, but that it doesn't have a object meaning, so I would suggest update it to something as 'creation_time' or a similar one that you would prefer.

Reply:
Renamed it "datePosted"